### DISCUSSION TITLE
3D Scalar

### DISCUSSION THREAD

**Discussion by lucas-wilkins** [bytedeco/javacv#1884] [Q&A] [answered]
Why isn't there a 3D Scalar constructor?

**Comment by saudet** [comment]
Could you be a bit more specific? Do you have some sample code in C++ that you would like to know how to translate?

**Comment by lucas-wilkins** [comment, marked as answer]
Ah, ok, there's no 3D Scalar in OpenCV either, I guess that explains it. I've only used python bindings before, and I was trying to make a blank NxMx3 and set the colour, I was a little confused about it requiring a scalar and there not being a 3D version (I found RGB though).

**Reply by saudet** [reply]
In that case, for a Mat with only 3 channels, it simply ignores the last value of a Scalar.
